examples/medplum-demo-bots/src/billing-bots/cms-1500.ts

```diff
@@ -0,0 +1,381 @@
+import {
+  BotEvent,
+  formatAddress,
+  formatCodeableConcept,
+  formatDate,
+  formatHumanName,
+  formatMoney,
+  formatQuantity,
+  MedplumClient,
+} from '@medplum/core';
+import {
+  Claim,
+  Coverage,
+  Device,
+  Organization,
+  Patient,
+  Practitioner,
+  PractitionerRole,
+  RelatedPerson,
+} from '@medplum/fhirtypes';
+
+export async function handler(medplum: MedplumClient, event: BotEvent<Claim>): Promise<string> {
+  const claim = event.input;
```

Comment:
Maybe creating a DocumentReference with Media and linking to Claim might be the right call here.  Not sure what your perspective is @rahul1